### ISSUE TITLE
Packwiz Integration

### ISSUE THREAD

**Issue by ryleu** [TLATER/nix-minecraft-servers#1]
You noted quite a few issues with Curseforge that the project Packwiz has already solved. Have you considered integration with that project? If not, I think it's worth looking into. https://github.com/packwiz

**Comment by TLATER** [comment]
Not really, it certainly seems interesting though, thanks for the pointer! I'm also very excited about the existence of modrinth, it addresses a *lot* of my issues with curseforge, hope it sees more widespread adoption in the future :)

My guess is that FODs with packwiz will end up being more painful than I'd like, as it will attempt to download *all* the mods instead of giving you per-mod downloads. But ultimately that may not be so bad, sharing store paths between mod packs likely isn't that important of a target.

**Comment by ryleu** [comment]
Yeah, the Modrinth API is super well-documented and well thought out.
